AttackItem.h

```diff
@@ -1 +1,12 @@
-#include "Item.h"
\ No newline at end of file
+
+#pragma once
+
+#include "Item.h"
+#include "Character.h"
+#include "Utility.h"
```

Comment:
You aren't using these here. include in the cpp.  forward declare any pointer types.
- [ ] revise.